kili/types.py

```diff
@@ -134,12 +134,31 @@ class ProjectUser(ProjectUserWithoutProject):
 
 
 @dataclass
-class User(UserWithoutProjectUsers):
+class UserWithoutApiKey(UserWithoutProjectUsers):
     """
     A wrapper for User GraphQL object.
     """
     projectUsers = ProjectUser
 
+@dataclass
+class ApiKey:
+    """
+    A wrapper for DatasetAsset GraphQL object.
+    """
+    createdAt = 'createdAt'
+    id = 'id'
+    key = 'key'
+    name = 'name'
+    revoked = 'revoked'
+    user = UserWithoutApiKey
+    userId = 'userId'
+
+@dataclass
+class User(UserWithoutApiKey):
+    """
+    A wrapper for User GraphQL object.
+    """
+    apiKeys = ApiKey
```

Comment:
apiKeys: list[ApiKey] would be better No ? It seems it is not handled in other types but it can be done https://docs.python.org/3/library/dataclasses.html

Reply:
Ok, it's how we build the function, I create a technical debt on this issue